frontend/src/components/SilvicultureSearch/Openings/AdvancedSearchDropdown/index copy.tsx

```diff
@@ -0,0 +1,312 @@
+    cuttingPermit: "",
+    grossArea: "",
+    timberMark: "",
+    status: null as string | null,
+    openingFilters: [] as string[], // Array to hold multiple checkbox selections
+    blockStatuses: [] as string[], // Array to hold multiple block statuses
+  });
+
+  const items = [
+    { text: "Option 1", value: "option1" },
+    { text: "Option 2", value: "option2" },
+    { text: "Option 3 - a disabled item", value: "option3", disabled: true },
+    { text: "Option 4", value: "option4" },
+    { text: "Option 5", value: "option5" },
+    { text: "Option 6", value: "option6" },
+    { text: "Option 7", value: "option7" },
+    { text: "Option 8", value: "option8" },
+  ];
+  
+
+  const handleFilterChange = (updatedFilters: Partial<typeof filters>) => {
+    const newFilters = { ...filters, ...updatedFilters };
+    setFilters(newFilters);
+    onSearch(newFilters); // Send the updated filters to the parent component
+  };
+
+  const handleCheckboxChange = (value: string, group: string) => {
+    const selectedGroup = filters[group as keyof typeof filters] as string[];
+    const updatedGroup = selectedGroup.includes(value)
+      ? selectedGroup.filter((item) => item !== value)
+      : [...selectedGroup, value];
+
+    handleFilterChange({ [group]: updatedGroup });
+  };
+
+  return (
+    <div className="advanced-search-dropdown">
+      <div className="container-fluid advanced-search-container p-32">
+        <div className="row pb-32">
+          <div className="group-1">
+            <CheckboxGroup
+              orientation="horizontal"
+              className="horizontal-checkbox-group"
+              legendText="Opening Filters"
+            >
+              <Checkbox
+                labelText={`Openings created by me`}
+                id="checkbox-label-1"
+                checked={filters.openingFilters.includes(
+                  "Openings created by me"
+                )}
+                onChange={() =>
+                  handleCheckboxChange(
+                    "Openings created by me",
+                    "openingFilters"
+                  )
+                }
+              />
+              <Checkbox
+                labelText={`Submitted to FRPA section 108`}
+                id="checkbox-label-2"
+                checked={filters.openingFilters.includes(
+                  "Submitted to FRPA section 108"
+                )}
+                onChange={() =>
+                  handleCheckboxChange(
+                    "Submitted to FRPA section 108",
+                    "openingFilters"
+                  )
+                }
+              />
+            </CheckboxGroup>
+          </div>
+        </div>
+
+        <div className="row mb-3">
+          <div className="col-6">
+            <Dropdown
+              id="orguni-dropdown"
+              titleText="Org unit"
+              items={items}
+              itemToString={(item: any) => (item ? item.text : "")}
+              onChange={(e: any) =>
```

Comment:
issue: most of these `any` generic type could be typed. Since we're working with ESlint checks and validations, we need to keep an eye on this things.